src/ecalc/libraries/libecalc/common/libecalc/core/graph_result.py

```diff
@@ -405,7 +425,7 @@ def get_asset_result(self) -> libecalc.dto.result.EcalcModelResult:
             id=asset.id,
             name=asset_node_info.name,
             component_level=asset_node_info.component_level,
-            componentType=asset_node_info.component_level,
+            componentType=asset_node_info.component_type,
```

Comment:
bugfix?

Reply:
Yes, removed it. Will include in new PR.